data/users/network/src/commonMain/kotlin/dev/burnoo/demo/listapp/data/users/network/api/UsersApi.kt

```diff
@@ -0,0 +1,59 @@
+package dev.burnoo.demo.listapp.data.users.network.api
+
+import com.github.michaelbull.result.Result
+import com.github.michaelbull.result.mapError
+import com.github.michaelbull.result.runCatching
+import dev.burnoo.demo.listapp.data.users.network.UsersNetworkDataSource
+import dev.burnoo.demo.listapp.data.users.network.model.NetworkError
+import dev.burnoo.demo.listapp.data.users.network.model.NetworkUser
+import dev.burnoo.demo.listapp.data.users.network.model.NetworkUsersResponse
+import io.ktor.client.HttpClient
+import io.ktor.client.call.body
+import io.ktor.client.engine.HttpClientEngine
+import io.ktor.client.plugins.ClientRequestException
+import io.ktor.client.plugins.ServerResponseException
+import io.ktor.client.plugins.contentnegotiation.ContentNegotiation
+import io.ktor.client.plugins.defaultRequest
+import io.ktor.client.request.get
+import io.ktor.client.request.header
+import io.ktor.serialization.kotlinx.json.json
+import kotlinx.serialization.json.Json
+
+const val BASE_PATH = "/data/v1"
```

Comment:
Is it public just for tests? You could move it into some namespace (maybe `UsersNetworkDataSource` just so it isn't a global constant.

Reply:
https://github.com/burnoo/DemoListApp/commit/b99c348afa2854c348cf9e795f2331ba3e0a09ef